hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsProperties.kt

```diff
@@ -33,6 +33,7 @@ data class HmppsSqsProperties(
     val accessKeyId: String = "",
     val secretAccessKey: String = "",
     val propagateTracing: Boolean = true,
+    val bucketName: String = "",
```

Comment:
There's a table in the readme to document all properties - we should add this `bucketName` property and make clear it's only for the large message publisher.